tensorboard/plugins/scalar/tf_scalar_dashboard/tf-scalar-dashboard.html

```diff
@@ -292,65 +294,13 @@
         return categorizeTags(runToTag, selectedRuns, tagFilter);
       },
-      _tagMetadata(runToTagInfo, runs, tag) {
-        /*
-         * Each run--tag combination may have its own display name and
-         * description, but we display just one chart per tag: this is a
-         * dimension mismatch. We reconcile this as follows:
-         *
-         *   - We only show a display name if all runs agree. Otherwise,
-         *     we use the tag name without the "/scalar_summary" suffix.
-         *
-         *   - If all runs agree on a description, we use it. Otherwise,
-         *     we concatenate all descriptions, annotating which ones
-         *     came from which run, and display them in a list.
-         */
-        let unanimousDisplayName = undefined;
-        const descriptionToRuns = {};  // from description to runs it appears in
+      _tagMetadata(runToTagsInfo, runs, tag) {
+        const runToTagInfo = {};
         runs.forEach(run => {
-          const info = runToTagInfo[run][tag];
-          if (unanimousDisplayName === undefined) {
-            unanimousDisplayName = info.displayName;
-          }
-          if (unanimousDisplayName !== info.displayName) {
-            unanimousDisplayName = null;
-          }
-          if (descriptionToRuns[info.description] === undefined) {
-            descriptionToRuns[info.description] = [];
-          }
-          descriptionToRuns[info.description].push(run);
+            runToTagInfo[run] = runToTagsInfo[run][tag];
         });
-        const displayName =
-          unanimousDisplayName !== null ?
-          unanimousDisplayName :
-          tag.replace(/\/scalar_summary$/, '');
-        const description = (() => {
-          const descriptions = Object.keys(descriptionToRuns);
-          if (descriptions.length === 0) {
-            return '';
-          } else if (descriptions.length === 1) {
-            return descriptions[0];
-          } else {
-            const items = descriptions.map(description => {
-              const runs = descriptionToRuns[description].map(run => {
-                const safeRun = run
-                  .replace(/</g, '&lt;')
-                  .replace(/>/g, '&gt;')  // for symmetry :-)
-                  .replace(/&/g, '&amp;');
-                return `<code>${safeRun}</code>`;
-              });
-              const joined = runs.length > 2 ?
-                (runs.slice(0, runs.length - 1).join(', ')
-                  + ', and ' + runs[runs.length - 1]) :
-                runs.join(' and ');
-              const runNoun = runs.length === 1 ? 'run' : 'runs';
-              return `<li><p>For ${runNoun} ${joined}:</p>${description}</li>`;
-            });
-            const prefix = '<p><strong>Multiple descriptions:</strong></p>';
-            return `${prefix}<ul>${items.join('')}</ul>`;
-          }
-        })();
-        return {displayName, description};
+        const defaultDisplayName = tag.replace(/\/scalar_summary$/, '');
```

Comment:
Maybe a comment noting why the replacement?

Reply:
Sure, if you like.

Reply:
LGTM! Yeah, I'm mainly wondering what are all the things we can do to help someone familiarize themselves with our system.

Reply:
You got it.